Lib9c/Action/ClaimStakeReward.cs

```diff
@@ -12,25 +16,131 @@
 namespace Nekoyume.Action
 {
     /// <summary>
-    /// Hard forked at https://github.com/planetarium/lib9c/pull/1371
+    /// Hard forked at https://github.com/planetarium/lib9c/pull/1458
     /// </summary>
     [ActionType(ActionTypeText)]
     public class ClaimStakeReward : GameAction, IClaimStakeReward, IClaimStakeRewardV1
     {
-        public const long ObsoletedIndex = 5_549_200L;
-        private const string ActionTypeText = "claim_stake_reward2";
+        private const string ActionTypeText = "claim_stake_reward4";
+
+        private const string StakeRegularRewardSheetV1Data =
```

Comment:
previous reward data should be shown before the specified block index in the client side, maybe it will be better to change it to public.